tests/test_global_mutual_information_loss.py

```diff
@@ -26,28 +26,28 @@
 
 EXPECTED_VALUE = {
     "xyz_translation": [
-        -1.5860259532928467,
-        -0.5957175493240356,
-        -0.3855515122413635,
-        -0.28728482127189636,
-        -0.23416118323802948,
-        -0.19534644484519958,
-        -0.17001715302467346,
-        -0.15043553709983826,
-        -0.1366637945175171,
-        -0.12534910440444946,
+        -1.5860257,
+        -0.62433463,
+        -0.38217825,
+        -0.2905613,
+        -0.23233329,
+        -0.1961407,
+        -0.16905619,
+        -0.15100679,
+        -0.13666219,
+        -0.12635908,
     ],
     "xyz_rotation": [
-        -1.5860259532928467,
-        -0.29977330565452576,
-        -0.18411292135715485,
-        -0.1582011878490448,
-        -0.16107326745986938,
-        -0.165723517537117,
-        -0.1970357596874237,
-        -0.1755618453025818,
-        -0.17100191116333008,
-        -0.17264796793460846,
+        -1.5860257,
+        -0.30265224,
+        -0.18666176,
+        -0.15887907,
+        -0.1625064,
+        -0.16603896,
+        -0.19222091,
+        -0.18158069,
+        -0.167644,
+        -0.16698098,
```

Comment:
difference introduced by the resampler floating points--

previous `(2/(d-1)*g+1)*(d-1)/2=g+(d-1)/2` vs now `((2/d*g+1)*d-1)/2=g+(d-1)/2`